src/cdk/testing/dom-helpers.ts

```diff
@@ -0,0 +1,9 @@
+export function getTextWithExcludedElements(element: Element, excludeSelector: string) {
```

Comment:
Can we ensure that this function is not exported through the public API?

Reply:
I think its not anymore, the api golds just need to be updated